CMakeLists.txt

```diff
@@ -169,18 +169,6 @@ if(ROBOTOLOGY_ENABLE_DYNAMICS)
   endif()
 endif()
 
-if(ROBOTOLOGY_ENABLE_DYNAMICS_FULL_DEPS)
-  find_or_build_package(manif)
-  find_or_build_package(qhull)
-  find_or_build_package(casadi)
-
```

Comment:
This part shouldn't be removed right?

Reply:
Sorry, I forgot to describe this in the PR. As we are including these as dependencies of blf, there is no need to also duplicate these in the main CMakeLists.txt . I realized this when I had two modified in two places to enable cppad on Windows.